spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextResourceLoaderAutoConfiguration.java

```diff
@@ -29,6 +30,7 @@
  * @author Agim Emruli
  */
 @Configuration
+@ConditionalOnAwsCloudEnvironment
```

Comment:
Right, that was my intention.
If we use properties like:
```
cloud.aws.region.static
cloud.aws.stack.name
```
we shouldn't require EC2 environment.